tendermint/Cargo.toml

```
@@ -53,10 +53,13 @@ sha2 = { version = "0.10", optional = true, default-features = false }
 k256 = { version = "0.13", optional = true, default-features = false, features = ["alloc", "ecdsa"] }
 ripemd = { version = "0.1.3", optional = true, default-features = false }
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+instant = { version = "0.1", features = [ "wasm-bindgen" ], optional = true}
```

We should define `wasm-bindgen` in a higher level

```suggestion
instant = { version = "0.1", optional = true}
```

---

Added `wasm-bindgen` feature to tendermint, which enables `instant/wasm-bindgen`